commons-geometry-spherical/src/test/java/org/apache/commons/geometry/spherical/twod/GreatCircleTest.java

```diff
@@ -664,7 +664,7 @@ void testHashCode() {
         Assertions.assertEquals(hash, a.hashCode());
 
         Assertions.assertNotEquals(hash, b.hashCode());
-        Assertions.assertNotEquals(hash, c.hashCode());
+        Assertions.assertEquals(hash, c.hashCode());
```

Comment:
If the two circles are the same then this should be tested using `Assertions.assertEquals`. If they are the same then the hash code **_must_** be the same. If the objects are not equal then the hash code may or may not collide; it doesn't matter.

Any test that uses not equals on hash codes for objects that are not equal can potentially fail due to random hash collision.